### DISCUSSION TITLE
[Action Required] MongoDB CRUD Service is now Open Source

### DISCUSSION THREAD

**Discussion by Edo784Gift** [mia-platform/community#196] [Q&A] [answered]
Hi,
I just received an email with this very same subject but it isn't clear if an action is required ...

Can someone clarify?

Thank you

**Comment by ThisIsDemetrio** [comment, marked as answer]
Hi, @Edo784Gift .

As the subject of the mail you have received, the CRUD Service has been moved to a public repository, and the source code can be found here: https://github.com/mia-platform/crud-service .

From there, you can look the code, follow the issues (which regards technical matter), see what are the latest versions available, starting from the latest (version 6.3.0).

Nothing really changes on the application itself, but the development of the application from now on will become more "transparent". Also the input of more people can give us more technical ideas about how to improve the CRUD Service in terms of new feature, performance, security.. you name it.

If you're using the CRUD Service docker image to build your application.. until now you're probably use the version 6.2.0 or older (whose image was located in the Mia Platform Nexus Repository). If you want to upgrade to the 6.3.0 (and future versions, when they're going to be available), you should update the docker image reference to the [ghcr.io/mia-platform/crud-service](https://ghcr.io/mia-platform/crud-service) or [docker.io/miaplatform/crud-service](https://hub.docker.com/r/miaplatform/crud-service).

Hope this helped.
|D|

EDIT: if you're using the console, I'm also adding an image that shows where to update the CRUD Service:
![console-image](https://user-images.githubusercontent.com/5429953/218041188-d4b78a84-ec67-4d93-85da-b9bb777381fa.png)
